packages/strapi-plugin-content-manager/admin/src/components/CustomTable/Row.js

```diff
@@ -88,6 +89,14 @@ function Row({ canDelete, canUpdate, isBulkable, row, headers }) {
   const { emitEvent } = useGlobalContext();
 
   const links = [
+    {
+      icon: canCreate ? <Duplicate fill="black" /> : null,
+      onClick: e => {
+        e.stopPropagation();
+
+        goTo(`create/clone/${row.id}`);
+      },
+    },
```

Comment:
Can you add a label or a title to display a tooltip please?

Reply:
I just saw that the links variable is used in the IconLinks component that is a Buffet component. Let's do that later.